contents/models.py

```diff
@@ -0,0 +1,42 @@
+from django.db import models
+
+class Content(models.Model):
+    name        = models.CharField(max_length=45)
+    created_at  = models.DateTimeField(auto_now_add=True)
+    updated_at  = models.DateTimeField(auto_now=True)
```

Comment:
```suggestion
    created_at  = models.DateTimeField(auto_now_add=True)
    updated_at  = models.DateTimeField(auto_now=True)
```

이런 반복적인 시간기록 요소를 작성한다면, 매번 선언하기 보다는 아래와 같이 추상화된 class를 만들고 `상속` 받아서 구현해 주세요.

```python
class TimeStampModel(models.Model):
    created_at  = models.DateTimeField(auto_now_add = True)
    updated_at = models.DateTimeField(auto_now = True)

    class Meta:
        abstract = True

```

만약 다른 앱에 있는 모델 클래스에서도 사용하고 싶다면 `core` app을 생성해서 core.models.py에 클래스 선언뒤 임포트 해주세요.

Reply:
넵 알겠습니다!!